src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -307,15 +307,15 @@ public virtual Task<IEnumerable<string>> EnumerateVersions(PackageURL purl, bool
         /// <param name="purl">The PackageURL to check.</param>
         /// <param name="useCache">If the cache should be checked for the existence of this package.</param>
         /// <returns>True if the package is confirmed to exist in the repository. False otherwise.</returns>
-        public virtual async Task<bool> PackageExists(PackageURL purl, bool useCache = true)
+        public virtual async Task<bool> PackageExistsAsync(PackageURL purl, bool useCache = true)
```

Comment:
Async marker is optional with virtual - it is just a compiler hint per this dicsussion: https://stackoverflow.com/questions/25015853/is-it-ok-to-have-virtual-async-method-on-base-class. 

But should be consistent in the file.  The other virtual methods are not marked async.

Reply:
I'll probably just remove this one then

Reply:
Wait, this one has to be marked `async` as it has `await EnumerateVersionsAsync(purl, useCache)` within it. That's why it has it now and the others dont.

Reply:
Ah that makes sense then.